apps/new-dealer/fields/index.js

```diff
@@ -1,6 +1,43 @@
 'use strict';
 
 module.exports = {
+  'company': {
+    mixin: 'radio-group',
+    validate: ['required'],
+    className: ['form-group'],
+    options: [{
+      value: 'true',
+      label: 'fields.company.options.true',
```

Comment:
ooh ooh - watch this space - PR in template mixins for this to be looked up magically!